src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax;
+
+import static core.basesyntax.RandomInt.getRandomInt;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomIndex = getRandomInt(FIGURE_COUNT);
```

Comment:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

    public Figure getRandomFigure() {
        int randomIndex = random.nextInt(FIGURE_COUNT);
```
dont use static method in RandomInt 
you can add private method here, but don't use static methods